config/api.yml

```diff
@@ -1754,3 +1754,30 @@
       :delete:
       - :name: delete
         :identifier: delete_arbitration_setting
+  :arbitration_rules:
+    :description: Arbitration Rules
+    :identifier: miq_arbitration_rules
+    :options:
+    - :collection
+    :verbs: *70174834084700
```

Comment:
You will need to rebase this with master to account for https://github.com/ManageIQ/manageiq/pull/10204. Apologies!